e2e/_suites/fleet/services_test.go

```diff
@@ -7,9 +7,11 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var testVersion = agentVersionBase
```

Comment:
Do you prefer consuming the variable there, or hardcoding the value directly here?

I see benefits in both approaches:
1. reusing production value: one single source of truth for the variable
2. hardcoding base version here, with same value as in the production code: it decouples tests from production, which I see an important point.